docs/content/latest/tools/superset.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Use Apache Superset with YugabyteDB YSQL
+headerTitle: Apache Superset
+linkTitle: Apache Superset
+description: Use Apache Superset to explore and visulize data in YugabyteDB.
+menu:
+  latest:
+    identifier: superset
+    parent: tools
+    weight: 2794
+isTocNested: true
+showAsideToc: true
+---
+
+[Apache Superset](https://superset.apache.org/) is a fast, lightweight and intuitive open-source data exploration and visualization tool that helps you query your data stored in YugabyteDB and visualize it using simple line charts to highly detailed geospatial charts.
+
+![Superset Dashboard](/images/develop/tools/superset/dashboard.png)
+
+## Before you begin
+
+Your YugabyteDB cluster should be up and running. If you're new to YugabyteDB, create a local cluster in less than five minutes following the steps in [Quick Start](../../quick-start/install). You can also get started with the free tier of [YugabyteDB Fully-Managed Cloud](https://www.yugabyte.com/cloud/). You also need to load a relevant database in your YugabyteDB for exploration and visualizaation. If using yugabyted, you can load the Northwind sample database with the ./bin/yugabyted demo connect command from your shell or you can follow [instructions here](https://docs.yugabyte.com/latest/sample-data/northwind/).
+
+You also need to install Apache Superset to explore and visualize your data. You can install Superset using [Docker Compose](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose) (recommended) or from scratch using [python install (PIP)](https://superset.apache.org/docs/installation/installing-superset-from-scratch).
+
+## Connecting Apache Superset with YugabyteDB
+
+After successful installation, launch Superset in the browser at http://<hostname/IP>:8088. In case of local installation on your laptop, navigate to localhost:8088 or 127.0.0.1:8088. Superset comes with standard PostgreSQL driver that also connects to YugabyteDB. You can also manually install [psycopg2 driver](https://www.psycopg.org/docs/) to connect to YugabyteDB.
+
+Follow these steps to connect Apache Superset to YugabyteDB:
+
+1. Navigate to Data → Databases → + Databases and choose "PostgreSQL" from the “Connect a database” menu as shown below.
```

Comment:
Don't use the arrows if possible. They won't render correctly on all browsers/machines. The usual convention is > instead.
```suggestion
1. Navigate to Data > Databases > + Databases, and choose "PostgreSQL" from the “Connect a database” menu.
```